Lib/sqlite3/__main__.py

```diff
@@ -48,17 +48,24 @@ def runsource(self, source, filename="<input>", symbol="single"):
         Return True if more input is needed; buffering is done automatically.
         Return False if input is a complete statement ready for execution.
         """
-        match source:
-            case ".version":
-                print(f"{sqlite3.sqlite_version}")
-            case ".help":
-                print("Enter SQL code and press enter.")
-            case ".quit":
-                sys.exit(0)
-            case _:
-                if not sqlite3.complete_statement(source):
-                    return True
-                execute(self._cur, source)
+        if not source or source.isspace():
+            return False
+        if source[0] == ".":
+            match source[1:].strip():
+                case "version":
+                    print(f"{sqlite3.sqlite_version}")
+                case "help":
+                    print("Enter SQL code and press enter.")
+                case "quit":
+                    sys.exit(0)
+                case _ as unknown:
+                    if unknown:
+                        print("Error: unknown command or invalid arguments:"
+                              f'  "{unknown}". Enter ".help" for help')
```

Comment:
Entering ".help" does not help, because it does not output the list of commands.